<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Test listener"
           tests="4"
           failures="1"
           errors="0"
           skipped="1"
           time="0,005">
  <properties />
            <testcase name="testTask4" time="0,000" classname="tSmirnova_hW_auto03.ListenerTask.ListenerTest">
                <failure type="java.lang.AssertionError"
                  message="null"
              >
<![CDATA[java.lang.AssertionError: null
  org.testng.Assert.fail(Assert.java:97)
  org.testng.Assert.fail(Assert.java:102)
  tSmirnova_hW_auto03.ListenerTask.ListenerTest.testTask4(ListenerTest.java:29)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.base/java.lang.reflect.Method.invoke(Method.java:564)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
  org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
  org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
  org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
  org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
  org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
  java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
  org.testng.TestRunner.privateRun(TestRunner.java:766)
  org.testng.TestRunner.run(TestRunner.java:587)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
  org.testng.SuiteRunner.run(SuiteRunner.java:286)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
  org.testng.TestNG.runSuites(TestNG.java:1039)
  org.testng.TestNG.run(TestNG.java:1007)
  com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
  com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:110)
]]>
      </failure>

          </testcase>
  
            <testcase name="testTask3" time="0.0" classname="tSmirnova_hW_auto03.ListenerTask.ListenerTest">
          <skipped />
    </testcase>
  
            <testcase name="testTask1" time="0,004" classname="tSmirnova_hW_auto03.ListenerTask.ListenerTest" />
                <testcase name="testTask2" time="0,000" classname="tSmirnova_hW_auto03.ListenerTask.ListenerTest" />
      
</testsuite>
